cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -420,6 +437,90 @@ Replaced by:
 ----
 WHERE NOT isEmpty([1, 2, 3])
 ----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+distance(n.prop, point({x:0, y:0})
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+point.distance(n.prop, point({x:0, y:0})
+----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+point({x:0, y:0}) <= point({x:1, y:1}) <= point({x:2, y:2})
+----
+a|
+Using inequality operators `<`, `<=`, `>`, and `>=` on spatial points is removed.
+Please instead use:
+[source, cypher, role="noheader"]
+----
+point.withinBBox(point({x:1, y:1}), point({x:0, y:0}), point({x:2, y:2}))
+----
+
+a|
+label:syntax[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+USING PERIODIC COMMIT ...
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+CALL {
+  ...
+} IN TRANSACTIONS
+----
+
+a|
+label:syntax[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+CREATE (a {prop:7})-[r:R]->(b {prop: a.prop})
+----
+a|
+Patterns where a variable introduced in the pattern is also referenced from the same pattern are no longer supported.
+
+a|
+label:syntax[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+CALL { RETURN 1 }
+----
+a|
+Unaliased expressions are no longer supported in subquery `RETURN` clauses. Replaced by:
+[source, cypher, role="noheader"]
+----
+CALL { RETURN 1 AS one }
+----
+
+a|
+label:syntax[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+MATCH (a) RETURN (a)--()
+----
+a|
+Pattern expressions for checking existence are no longer supported, except for use in `WHERE` clauses. Use a pattern comprehension instead. Replaced by:
```

Comment:
I intentionally left out `and in exists()` from this table, as it is for 5.0 where `exists()` has been removed. But I will update it to something similar as your suggestion

Reply:
I think you have a point.

You could consider writing
> as existence predicates, **for example** in `WHERE` clauses.
as pattern expressions can also be used in other boolean contexts such as `CASE` statements.

Reply:
Also, from an English point of view, I would expect either

> Instead, use **a** pattern comprehension:

or

> Instead, use pattern comprehension**s**:

Same applies to the section deprecated in 4.3.

Reply:
You are absolutely right. I just left the way it was because I thought it could be used as a more generic technical term, but I guess adding an "a" will make the sentence better. Thanks for flagging this!